trillian/ctfe/ct_server/main.go

```diff
@@ -195,8 +196,13 @@ func main() {
 	go awaitSignal(func() {
 		os.Exit(1)
 	})
-	server := http.Server{Addr: *httpEndpoint, Handler: nil}
-	err = server.ListenAndServe()
+
+	// Wrap DefaultServeMux with a CORS handler that allows cross-origin
+	// requests. This is safe because CT logs are public and unauthenticated
+	// so cross-site scripting attacks are not a concern.
+	corsHandler := cors.AllowAll().Handler(http.DefaultServeMux)
```

Comment:
`AllowAll` seems kinda broad -- is it possible to restrict to just the `<log>/ct/v1/...` paths, maybe by tweaking something in `setupAndRegister` below?

Reply:
I've restricted it to just whatever handlers are setup by `ctfe.SetUpInstance()` and "/".